addons/mrp/static/src/product_catalog/mrp_move_raw/mrp_move_raw.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-name="ProductCatalogMrpMoveRawLine"
+       t-inherit="product.ProductCatalogOrderLine"
+       t-inherit-mode="primary">
+        <xpath expr="//i[hasclass('fa fa-shopping-cart')]" position="attributes">
+             <attribute name="class">fa fa-gear</attribute>
```

Comment:
We really do an overide for that

Reply:
Hello @amoyaux,

As per the spec, we have only to do this for mrp and not for all. Therefore, I need to overridden this template accordingly.

Thanks.